webpack.config.js

```diff
@@ -62,6 +62,7 @@ function createConfig(isProdBuild, latestBuild) {
     copyPluginOpts.push({ from: 'build-translations/output', to: `translations` });
     copyPluginOpts.push({ from: 'node_modules/@polymer/font-roboto-local/fonts', to: 'fonts' });
     copyPluginOpts.push('node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js')
+    copyPluginOpts.push({ from: 'node_modules/react-big-calendar/lib/css/react-big-calendar.css', to: `images/react-big-calendar/` });
```

Comment:
Feels weird to put it in an images folder

Reply:
Do you want I create a css folder ?

Reply:
Maybe make a panel based folder ?

Reply:
Done !